### DISCUSSION TITLE
Ability to show messages that are ignored by patterns

### DISCUSSION THREAD

**Discussion by x9136** [Chatterino/chatterino2#5204] [Ideas]
I suggest adding an option so that ignored messages can be replaced with a "Show" button. In theory, this would be another column with a checkbox in the pattern table. I create ignore patterns for messages that take up 8+ lines of chat and are often repeated, but sometimes it makes sense to take a quick look at them. 

It's also useful against bots, for example to make a pattern to hide repetitive ads or messages from integrations with games. Sometimes you get the urge to see that messages, though mostly you don't. You might want to think about the implementation, because there is an option to associate the idea with profiles, but in that case you'd have to always open the bot's responses to commands. So I would prefer to associate the idea with patterns only.

**Comment by pajlada** [comment]
Essentially, this would replace the "Block" column with a dropdown of what to do with the ignored message
1) Replace matching pattern (current behaviour if the Block column is unchecked)
2) Block message (current behaviour if the Block column is checked)
3) Censor message (which would censor it, maybe blur it or something, where if you click it it's permanently uncensored)

something that like?

**Comment by Mm2PL** [comment]
This is somewhat similar to https://github.com/Chatterino/chatterino2/issues/5217. Maybe we could cleverly merge these two ideas to be the same system.
